client/src/server.rs

```diff
@@ -141,15 +176,15 @@ impl Server for ServerAPI {
         self.make_request(ReqType::Delete(payload), uri).await
     }
 
-    fn new() -> Result<ServerAPI, Box<dyn std::error::Error>> {
-        let test_client = Config::new()
-            .set_base_url(Url::parse("http://127.0.0.1:50051")?)
-            .set_timeout(Some(Duration::from_secs(5)))
-            .try_into()?;
-
-        Ok(ServerAPI {
-            client: test_client,
-        })
+    fn new() -> Self {
+        let server_url = Url::parse(SERVER_URL).expect("Couldn't parse server URL");
+        ServerAPI {
+            client: Config::new()
+                .set_base_url(server_url)
+                .set_timeout(Some(Duration::from_secs(5)))
+                .try_into()
```

Comment:
Bruger vi try_into i stedet for into bare for at få custom panic message med expect?